src/Npgsql/NpgsqlActivitySource.cs

```diff
@@ -49,6 +49,9 @@ static class NpgsqlActivitySource
 
         activity.SetTag("db.query.text", commandText);
 
+        if (prepared is true)
```

Comment:
Note that I'm not setting db.npgsql.prepared to false, but rather omitting it. I think having a false vs. absent distinction isn't an important thing in this context, and it's probably better to just save on the useless false tag (but we could also go the other way).